cpg_infra/billing_aggregator/aggregate/seqr.py

```diff
@@ -443,14 +443,15 @@ def generator() -> Generator[dict, None, None]:
             obj['id'] = nid
 
             # For every seqr billing entry migrate it over
-            entries.append(obj)
-            entries.append(
-                utils.get_credit(
-                    entry=obj,
-                    topic='seqr',
-                    project=utils.SEQR_PROJECT_FIELD,
-                ),
+            entries.append(obj) if obj['id'] not in existing_ids else None
```

Comment:
```suggestion
            if obj['id'] not in existing_ids:
                entries.append(obj)
```

Reply:
Yes, that one was my original, than I did not like the main loop does not fit on the screen.
But I guess it reads better.